gateway-developer-plugin/src/test/java/com/ca/apim/gateway/capublisherplugin/CAGatewayDeveloperTest.java

```diff
@@ -67,6 +67,12 @@ void testExampleProject(TemporaryFolder temporaryFolder) throws IOException, URI
 
         File buildDir = new File(testProjectDir, "build");
         validateBuildDir(projectFolder, buildDir);
+        File buildGatewayDir = new File(buildDir, "gateway");
+        File buildGatewayBundlesDir = new File(buildGatewayDir, "bundle");
+        File builtBundleFile = new File(buildGatewayBundlesDir,  projectFolder + projectVersion + ".bundle");
+        assertTrue(builtBundleFile.isFile());
+        File builtDeleteBundleFile = new File(buildGatewayBundlesDir,  projectFolder + projectVersion + ".delete.bundle");
```

Comment:
Use `DELETE_BUNDLE_EXTENSION` instead of string constant

Reply:
done